assets/charts/components/inspektor-gadget/values.yaml

```diff
@@ -0,0 +1,42 @@
+# Default values for inspektor-gadget.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: kinvolk/gadget
```

Comment:
Unrelated but we should also push it to quay

Reply:
Filed an issue upstream: https://github.com/kinvolk/inspektor-gadget/issues/141

Reply:
cool